src/adhocracy_euth/adhocracy_euth/resources/collaborative_text.py

```diff
@@ -13,6 +13,17 @@ class IProcess(document_process.IDocumentProcess):
 )
 
 
+class IPrivateProcess(document_process.IDocumentProcess):
+    """Private collaborative text participation process."""
+
```

Comment:
IPrivateProcess should subclass IProcess, it is actually the same process type exept different permisions.